en/docs/design/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes.md

```diff
@@ -243,5 +243,45 @@ When a scope is attached to an API resource, access to it gets restricted based
 
      If you first create a local scope and then create a shared scope with same scope name before attaching the local scope to any API resource, the local scope will removed from local scope UI. However, it will not be reflected in the API Definition, unless you save the API. This is an identified limitation in supporting both local and shared scopes.
 
+## Restrict Unassigned Scopes
```

Comment:
Let's make the same changes requested in [1] here.

[1] https://github.com/wso2/docs-apim/pull/8090